examples/demo4.yaml

```diff
@@ -103,14 +106,32 @@ gal :
     type : Sum
 
     items :
+        # 0.6 of flux is in the total disk.  Further divide this
+        # into a smooth disk and knots of star formation, modeled
+        # as point sources distributed via a random walk
+        #
+        # note the knots get the same hlr as the smooth disk, and
+        # get sheared by the same ellipticity, but the knots will
+        # themselves have additional shape noise
+
         - 
             type : Exponential
             half_light_radius : { type : Catalog , col : 5 }
             ellip : 
                 type : E1E2 
                 e1 : { type : Catalog , col : 6 }  
                 e2 : { type : Catalog , col : 7 }
-            flux : 0.6
+            flux : 0.18
```

Comment:
I would think you could use the config eval functionality to define these in terms of each other (i.e., define variables for a bulge fraction and knot fraction, then use those to set the fluxes of each component using the appropriate formula) rather than having them be hard-coded numbers.